packages/aztec.js/src/encoder/outputCoder.js

```diff
@@ -64,16 +81,26 @@ outputCoder.decodeNote = (note) => {
  *
  * @method decodeNotes
  * @param {note} notes - array of AZTEC notes
+ * @param {bool} isOutputNote - boolean determining whether the supplied notes are outputNotes.
+ * If they are, decodeOutputNote() is called rather than decodeInputNote()
  * @returns {Object[]} array of note variables - array of decoded and extracted note variables
  * where each element corresponds to the note variables for an individual note
  */
-outputCoder.decodeNotes = (notes) => {
+outputCoder.decodeNotes = (notes, isOutputNote) => {
```

Comment:
This random bool feels like it shouldn't be there, we have enough information in the note to derive it without require the developer to input it.

Reply:
Removed it